nupic/algorithms/anomaly.py

```diff
@@ -157,6 +144,30 @@ def compute(self, activeColumns, predictedColumns,
     return score
 
 
+############################################
+# implementations of compute() function
+  def computeRaw(self, active, prevPredicted, inputValue=None, timestamp=None):
```

Comment:
`computeRaw`, `computeLikelihood`, and `computeWeighted` are private methods and must have a leading underscore in their names; e.g., `_computeWeighted`